redbot/cogs/mod/events.py

```diff
@@ -72,6 +71,60 @@ async def check_mention_spam(self, message):
                         print(e)
                         return False
                     return True
+
+        if mention_spam["kick"]:
+            if len(mentions) >= mention_spam["kick"]:
+                try:
+                    await guild.kick(author, reason=_("Mention Spam (Autokick)"))
+                except discord.HTTPException:
+                    log.info(
```

Comment:
I'm not sure if this log level should be `info`, this needs some discussion.

I understand this and the two suggestions below reference code modeled after existing code, however the existing code should be changed as well.

Reply:
Changed to `log.warning` not entirely sure what else to go for here. Willing to change this type of logging if anyone has something to suggest.